src/staticdata/staticdata.ts

```diff
@@ -114,6 +134,18 @@ class StaticData {
     };
 
     /**
+     * Update accommodationsEquipment information.
+     *
+     * @param accommodationsEquipmentId:string
+     * @param rev:string
+     * @param accommodationsEquipment:accommodationsEquipment
+     * @param callback
+     */
+    updateAccommodationEquipment = (accommodationsEquipmentId:string, rev:string, accommodationsEquipment, callback) => {
+        this.db.save(accommodationsEquipmentId, rev, accommodationsEquipment, callback);
```

Comment:
It is both possible! But it makes no difference, because we use Swagger to update the accommodation equipment database objects.